src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/GreaterThanInstruction.cs

```diff
@@ -323,320 +323,4 @@ public static Instruction Create(Type type, bool liftedToNull = false)
 
         public override string ToString() => "GreaterThan()";
     }
-
-    internal abstract class GreaterThanOrEqualInstruction : Instruction
-    {
-        private readonly object _nullValue;
-        private static Instruction s_SByte, s_int16, s_char, s_int32, s_int64, s_byte, s_UInt16, s_UInt32, s_UInt64, s_single, s_double;
-        private static Instruction s_liftedToNullSByte, s_liftedToNullInt16, s_liftedToNullChar, s_liftedToNullInt32, s_liftedToNullInt64, s_liftedToNullByte, s_liftedToNullUInt16, s_liftedToNullUInt32, s_liftedToNullUInt64, s_liftedToNullSingle, s_liftedToNullDouble;
-
-        public override int ConsumedStack => 2;
-        public override int ProducedStack => 1;
-        public override string InstructionName => "GreaterThanOrEqual";
-
-        private GreaterThanOrEqualInstruction(object nullValue)
-        {
-            _nullValue = nullValue;
-        }
-
-        private sealed class GreaterThanOrEqualSByte : GreaterThanOrEqualInstruction
-        {
-            public GreaterThanOrEqualSByte(object nullValue)
-                : base(nullValue)
-            {
-            }
-
-            public override int Run(InterpretedFrame frame)
-            {
-                object right = frame.Pop();
-                object left = frame.Pop();
-                if (left == null || right == null)
-                {
-                    frame.Push(_nullValue);
-                }
-                else
-                {
-                    frame.Push(((sbyte)left) >= (sbyte)right);
```

Comment:
Notice there are some unnecessary parentheses here. Code generation for these instruction will omit these. I'll submit a PR to remove these after this one goes through, so we reduce the diff when switching to T4 generation.